tests/txpool/pool_test.go

```diff
@@ -0,0 +1,169 @@
+//go:build integration
+
+package txpool
+
+import (
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/erigontech/erigon-lib/crypto"
+	"github.com/erigontech/erigon-lib/gointerfaces/sentryproto"
+	"github.com/erigontech/erigon-lib/log/v3"
+	"github.com/erigontech/erigon/cmd/devnet/requests"
+	"github.com/erigontech/erigon/core/types"
+	"github.com/erigontech/erigon/tests/txpool/helper"
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	// addr1 = 0x71562b71999873DB5b286dF957af199Ec94617F7
+	pkey1, _ = crypto.HexToECDSA("26e86e45f6fc45ec6e2ecd128cec80fa1d1505e5507dcd2ae58c3130a7a97b48")
+	addr1    = crypto.PubkeyToAddress(pkey1.PublicKey)
+
+	// addr2 = 0x9fB29AAc15b9A4B7F17c3385939b007540f4d791
+	pkey2, _ = crypto.HexToECDSA("45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8")
+	addr2    = crypto.PubkeyToAddress(pkey2.PublicKey)
+
+	rpcAddressNode1 = "localhost:8545"
+	rpcAddressNode2 = "localhost:8546"
+)
+
+// Topology of the network:
+// p2p_helper ---> node1 <--- RPC txns
+//
+// This test sends transaction to node1 RPC which means they are local for node1
+// P2P helper is binded to node1 port, that's why we measure performance of local txs processing
+func TestSimpleLocalTxThroughputBenchmark(t *testing.T) {
```

Comment:
please use Golang's built-in benchmark feature: https://gobyexample.com/testing-and-benchmarking